super-csv/src/main/java/org/supercsv/io/AbstractCsvWriter.java

```diff
@@ -64,26 +63,26 @@ public AbstractCsvWriter(final Writer writer, final CsvPreference preference) {
 		} else if( preference == null ) {
 			throw new NullPointerException("preference should not be null");
 		}
-		
-		this.writer = new BufferedWriter(writer);
```

Comment:
This change brakes backward compatibility. I think that we should add new constructor which takes additional constructor.